guides/common/modules/proc_creating-a-remote-source-for-a-custom-file-type-repository.adoc

```diff
@@ -39,20 +39,19 @@ ifdef::satellite[]
 # subscription-manager repos \
 --enable={RepoRHEL8AppStream} \
 --enable={RepoRHEL8BaseOS} \
---enable={RepoRHEL8ServerSatelliteUtils}
-----
-. Enable the satellite-utils module:
-+
-[options="nowrap", subs="+quotes,verbatim,attributes"]
-----
-# dnf module enable satellite-utils
+--enable={RepoRHEL8ServerSatelliteServerProjectVersion}
 ----
 endif::[]
 . Install the Pulp Manifest package:
 +
-[options="nowrap" subs="+quotes,attributes"]
+[options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# dnf install pulp-manifest
+ifdef::satellite[]
+# {project-package-install} python3.11-pulp_manifest
```

Comment:
Thanks for the screenshots, I see now that this should just be `dnf` instead of `satellite-maintain packages install`. Satellite wouldn't be installed on a remote system, which negates the need for satellite-maintain to install packages.